api/requirements.in

```diff
@@ -28,3 +28,6 @@ gunicorn
 
 # doc-generation
 drf-spectacular
+
+# django-jquery
+django-jquery
```

Comment:
I think it can just be moved up with the rest of the Django apps
Also, run `make requirements-build` to also build the `requirements-dev.txt` file